src/vcpkg/commands.build.cpp

```diff
@@ -1593,18 +1624,19 @@ namespace vcpkg
                                                         const Optional<Path>& issue_body)
     {
         const auto& spec_name = action.spec.name();
+        const auto& triplet_name = action.spec.triplet().to_string();
```

Comment:
Can we pass around `const Triplet&` rather than unwrapping it like this? `Triplet`'s whole reason to be is to avoid such copies.

Reply:
But here is no copy?

Reply:
:O

Oh